app.js

```diff
@@ -78,9 +87,59 @@ const STREAM_WORST_QUALITY_QUERY =
   '[data-a-target="player-settings-menu"] .tw-pd-05:last-child .tw-radio';
 const CHANNEL_STATUS_QUERY = ".tw-channel-status-text-indicator";
 const NO_DROPS_QUERY = 'div[data-test-selector="drops-list__no-drops-default"]';
+const DROP_INVENTORY_LIST = 'div.tw-flex-wrap.tw-tower.tw-tower--180.tw-tower--gutter-sm';
+const DROP_ITEM = '.tw-flex';
+const CATEGORY_NOT_FOUND = '[data-a-target="core-error-message"]';
+const DROP_STATUS = '[data-a-target="Drops Enabled"]';
 
 // ========================================== CONFIG SECTION =================================================================
 
+// ========================================== UTILS SECTION =================================================================
+/**
+* @fn idle
+* @param (ms) number of milliseconds to wait until the program resumes execution 
+* @author AlexSimpler
+* @return a promise which resolves after a timeout
+*/
+function idle(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+/**
+* @fn upperFirst
+* @param (word) makes the first letter of the word uppercase
+* @author AlexSimpler
+* @return the modified word
+*/
+function upperFirst(word) {
+  return (word[0].toUpperCase() + word.substring(1));
+}
+
+/**
+* @fn getUserProperty (Beta)
+* @param (page, name) takes as parameters the page handler aswell as the name of the property inside twilight-user
+* @author AlexSimpler
+* @return the value of the cookie
+*/
+async function getUserProperty(page, name) {
+
+  if (!name || !(/^[A-Za-z1-9]+$/.test(name))) throw new Error("Invalid cookie name: ", name);
+
+  const data = await page.cookies();
+  let cookieValue = undefined;
+
+  for (let i = 0; i < data.length; i++) {
+    if (data[i].name == 'twilight-user') {
+      cookieValue = JSON.stringify((data[i].value).replace(/\%+[1-9]+/gm, ' ').replace(/\ \C\ /gm, ""));
+      cookieValue = cookieValue.replace(/"+/gm, "");
+      let reg = new RegExp(`(?<=${name}\\s\\:\\s)[a-zA-Z0-9]+`, 'gm');
+      cookieValue = cookieValue.match(reg);
+    }
+  }
+  return cookieValue ? cookieValue[0] : new Error("Invalid cookie returned");
```

Comment:
Why do we return an error instead of throwing it?

Reply:
Because i can't throw in a return statement.